<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21_credit</class>
 <widget class="QWidget" name="s21_credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #00A876;</string>
  </property>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>489</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-size: 30px;
	color: white;
	background-color: #F2A918;
	border-style: outset;
	border-width: 2px;
	border-color: beige;
	border-radius: 30px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>341</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-size: 30px;
	color: white;
	background-color: #F2A918;
	border-style: outset;
	border-width: 2px;
	border-color: beige;
	border-radius: 30px;
}</string>
   </property>
   <property name="text">
    <string>CALCULATE</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_diff">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
color: rgb(0, 0, 0);
font-size: 25px;
}</string>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_ann">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QRadioButton {
color: rgb(0, 0, 0);
font-size: 25px;
}</string>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Credit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Credit term</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Payment type</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_rate">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000099.989999999990687</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin_amount">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000099.989999999990687</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_term">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>580</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}
</string>
   </property>
   <property name="text">
    <string>Monthly payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>580</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>580</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}
</string>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_over">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>640</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>640</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>700</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}
</string>
   </property>
   <property name="text">
    <string>Total payout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>700</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>700</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_diff">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}

</string>
   </property>
   <property name="text">
    <string>Number month</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;/Users/lymondgl/Desktop/s21_SmartCalc-v2.0/src/images/sber.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>Months</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 25px;
	color: rgb(0, 0, 0);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-top-right-radius: 20px;
	border-bottom-right-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
